packages/core/src/Stores/client-store.js

```diff
@@ -505,13 +505,32 @@ export default class ClientStore extends BaseStore {
     }
 
     get can_change_fiat_currency() {
-        const has_no_mt5 = !this.has_real_mt5_login;
-        const has_no_dxtrade = !this.has_real_dxtrade_login;
+        const eu_user = this.root_store.traders_hub.is_eu_user;
+
+        const has_maltainvest_mt5_account = this.mt5_login_list.some(
+            acc => acc.landing_company_short === 'maltainvest'
+        );
+        const has_svg_and_financial_mt5_account = this.mt5_login_list.some(
+            acc => acc.landing_company_short === 'svg' || acc.landing_company_short === 'financial'
```

Comment:
do we have `landing_company_short === 'financial'` ?

Reply:
we do not have such landing_comapny_short_code. we have "financial"  market_type

Reply:
we have "svg", "bvi", "labuan", "vanuatu" as shortcodes for landing_companies.

Reply:
Noted @hirad-rewok. Will make the changes 👀